doc/requirements.txt

```
@@ -1,7 +1,7 @@
 # Requirements to test and build the documentation
 # Install via pip: sudo pip install -U -r requirements.txt
-sphinx >= 1.6
-matplotlib>=2.0
+sphinx == 1.6.1
```

I am not a fan of hardcoding a specific Sphinx version. The website is build with 1.7.5 (on g4), and I have no problems locally with 1.8.2. Can we agree on `>=`?

```suggestion
sphinx >= 1.6.1
```